third_party/proton/test/test_instrumentation.py

```diff
@@ -164,14 +170,15 @@ def add_kernel(
     output = torch.empty_like(x)
     n_elements = output.numel()
     grid = (1, 1, 1)
-    proton.start(str(temp_file.with_suffix("")), backend="instrumentation")
+    proton.start(str(temp_file.with_suffix("")), backend="instrumentation", hook="launch")
     # cycle values are aggregated from all warps, set num_warps=1 to just
```

Comment:
Is this still true? I think you added the normalization right?

Reply:
Yes, it's fine as we don't check the number of aggregated cycles